lib/internal/cluster/master.js

```diff
@@ -115,7 +115,7 @@ function createWorkerProcess(id, env) {
 
   for (var i = 0; i < execArgv.length; i++) {
     const match = execArgv[i].match(
-      /^(--inspect|--inspect-(brk|port)|--debug|--debug-(brk|port))(=\d+)?$/
+      /^(--inspect|--inspect-(brk|port)|--debug|--debug-(brk|port))(=.*)?$/
```

Comment:
This will match flags that specify a host, but it seems like the host would then be lost when the flag is rewritten for the worker process a few lines down.